sqlglot/dialects/presto.py

```diff
@@ -173,6 +173,29 @@ def _unix_to_time_sql(self: Presto.Generator, expression: exp.UnixToTime) -> str
     return f"FROM_UNIXTIME(CAST({timestamp} AS DOUBLE) / POW(10, {scale}))"
 
 
+def _jsonextract_sql(self: Presto.Generator, expression: exp.JSONExtract) -> str:
+    is_json_extract = self.dialect.settings.get("VARIANT_EXTRACT_IS_JSON_EXTRACT", True)
```

Comment:
let's not do uppercase, it's inconsistent

Reply:
also `prop=true` in the dialect settings looks a bit verbose, should we just update the setting parsing logic so you can do something like `dialect=trino,normalization_strategy=case_sensitive,variant_extract_is_json_extract` and then just do `dialect.settings.get("variant_extract_is_json_extract")`? cc @tobymao

Reply:
I agree, not only that but the kwarg parsing turns the RHS values to strings so you also have to convert `'True'` to bool `True` before checking the condition, making it even more verbose. 

Should we then add the following in the `settings` parsing logic?
1. If a setting does not have a value, insert `True`
2. If a setting has the `str` value  `'True'` / `'False'` and swap out the `str` value to a `bool` one

Reply:
Sure, I like that, let's see what Toby thinks.

Reply:
you should be able to use any truthy value

Reply:
Was experimenting with this, haven't found a combination of built-in functions which perfectly evaluate strings to bools. Is it fine to make a dict of accepted truthy values (e.g. "true",  "t", non-zero integer strings etc) and similarly one for falsy ones?

Reply:
See https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/utils/__init__.py#L190-L200

Reply:
let's only support 1, 0, true, false